README.md

```diff
@@ -27,13 +27,17 @@ We provide examples in Swift and Objective-C. Run `carthage update --platform io
 
 #### [CocoaPods](https://cocoapods.org/)
 
-You'll need to install two pods, `MapboxNavigation.swift` and `MapboxDirections.swift`
-
 Add the following lines to your Podfile:
 
 ```ruby
-pod 'MapboxDirections.swift', :git => 'https://github.com/mapbox/MapboxDirections.swift.git', :commit => 'ceaf58b780fc17ea44a9150041b602d017c1e567'
-pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.0.4'
+# Required
+pod 'MapboxDirections.swift', :git => 'https://github.com/mapbox/MapboxDirections.swift.git', :commit => '955970285f7dee31b2c130023d7b383be51ab190'
+pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'c5a637a201a3043c24e52f3e2c50a1657a8ffced'
```

Comment:
Actually, that’s true about an application’s Podfile, but since this is a transitive dependency, you’re right that we need to specify either a tag or commit here. Just rename `tag` to `commit`.